source/manage/mmctl-command-line-tool.rst

```diff
@@ -2405,7 +2408,7 @@ mmctl export job
 
 **Description**
 
-List and show export jobs.
+List, show, and export jobs.
```

Comment:
This edit leads me to believe the commands are about ```jobs``` and not ```exports```. Can you confirm?